client/app/visualizations/edit-visualization-dialog.js

```diff
@@ -32,6 +38,24 @@ const EditVisualizationDialog = {
       this.visualization = this.newVisualization();
     }
 
+    this.has3plusColumnsFunction = () => {
+      let has3plusColumns = false;
+      if ((JSON.stringify(this.visualization.options.columnMapping).match(/,/g) || []).length > 1) {
```

Comment:
Why this and not [_.size](https://lodash.com/docs/4.17.11#size)?

Reply:
Great point - a way easier read. My brain just got stuck on it being a string.